File: 02.js
const lados = Number(prompt('Cuántos lados tiene la figura? '));
const longitud = Number(prompt('Cuánto mide cada lado? '));
const apotema = Number(prompt('Cuánto mide el apotema? '));
let perimetro = lados * longitud;
let area = ((perimetro * apotema) / 2).toFixed(2);

if (lados == 3) {
  console.log(
    `Es un triángulo con un perímetro de ${perimetro} y un área de ${area}`
  );
} else if (lados == 4) {
  console.log(
    `Es un cuadrado con un perímetro de ${perimetro} y un área de ${area}`
  );
} else if (lados == 5) {
  console.log(
    `Es un pentágono con un perímetro de ${perimetro} y un área de ${area}`
  );
} else {
  console.log(
    `Es un polígono con ${lados} lados, un perímetro de ${perimetro} y un área de ${area}`
  );
}

File: 02.py
def area_figura():
    lados = float(input("¿Cuántos lados tiene la figura? "))
    longitud = float(
        input("¿Cuánto mide la medida uno de los lados de la figura? "))
    apotema = float(input("¿Cuánto mide la apotema de la figura? "))
    perimetro = lados * longitud
    area = (perimetro * apotema) / 2
    if lados == 3:
        print("El área del triángulo es: ", area)
    elif lados == 4:
        print("El área del cuadrado es: ", area)
    elif lados == 5:
        print("El área del pentágono es: ", area)
    else:
        print(f"Es un polígono con {lados} y un área de: {round(area)}.")


area_figura()
